swift_browser_ui_frontend/src/views/ShareRequests.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-  <section>
+  <div class="contents">
```

Comment:
`section` would have worked as well because it is not related to flow but more towards sectioning content, see:  https://html.spec.whatwg.org/multipage/indices.html#element-content-categories

Reply:
Yup, seems to be so as I read more into it, I'll revert back to section as it's more descriptive